tests/cframework/tests.c

```diff
@@ -513,7 +539,8 @@ static void clean_temp_home (void)
 		rmdir (fileToClean);
 		elektraFree (fileToClean);
 
-		rmdir (tempHome);
+		succeed_if (rmdir (tempHome) == 0, "Could not delete TMPHOME manually");
```

Comment:
won't this always fail? (Our tests are not clean at the moment)

Reply:
If you run gpg tests on a system with gpg2 instead of gpg that has no nftw exported, probably.

Reply:
I am in favor of removing this `succeed_if` and instead add a build-server check if all tmp files are cleaned up in the build jobs. What do you think? (see #287)